iothub_client/src/iothub_client_authorization.c

```diff
@@ -676,6 +678,12 @@ int IoTHubClient_Auth_Set_SasToken_Expiry(IOTHUB_AUTHORIZATION_HANDLE handle, si
         LogError("Invalid handle value handle: NULL");
         result = __FAILURE__;
     }
+    // Validate the expiry_time in seconds
+    else if (expiry_time_seconds < MIN_SAS_EXPIRY_TIME)
+    {
+        LogError("Failure setting expiry time below the min value of %lu", (unsigned long)expiry_time_seconds);
```

Comment:
>lu [](start = 70, length = 2)

You need to change this to say "this is actual; value" and "you need to set it to something greater than MIN_SAS_EXPIRY_TIME"